app/models/user.rb

```diff
@@ -131,6 +132,8 @@ class User < ApplicationRecord
 
   scope :with_data, -> { joins(:data) }
   scope :donor, -> { with_data.where.not(user_data: { first_donated_at: nil }) }
+  scope :premium, -> { with_data.where('user_data.premium_until > ?', Time.current) }
+  scope :insiders, -> { with_data.where(user_data: { insiders_status: %i[active active_lifetime] }) }
```

Comment:
We need indexes on both of these columns.

Reply:
They were already there:

https://github.com/exercism/website/blob/main/db/migrate/20230412075814_add_insiders_status_to_users.rb#L6
https://github.com/exercism/website/pull/5467/files#diff-b12c7e45b80830692065f74867506a622dbe92f05ac9099de2ff63015d6808b6R6